### ISSUE TITLE
addAlternativePhoneMasks not working

### ISSUE THREAD

**Issue by karabanovbs** [caseyryan/flutter_multi_formatter#40]
Hi, I'm trying to add an alternative mask for New Zealand.

```
    PhoneInputFormatter.addAlternativePhoneMasks(
      countryCode: 'NZ',
      alternativeMasks: [
        '+00 (0) 000 0000',
        '+00 (00) 000 0000',
        '+00 (000) 000 0000',
      ],
    );
```

But it is not working. Log:
`Alternative masks for the country "New Zealand" is now null`

Why [this](https://github.com/caseyryan/flutter_multi_formatter/blob/4684de8e7568f895e052d1facc55429253a7e521/lib/formatters/phone_input_formatter.dart#L175) check needs?

**Comment by karabanovbs** [comment]
This works for me.
```
    PhoneInputFormatter.replacePhoneMask(
        countryCode: 'NZ', newMask: '+00 (000) 000 0000');
    PhoneInputFormatter.addAlternativePhoneMasks(
      countryCode: 'NZ',
      alternativeMasks: [
        '+00 (0) 000 0000',
        '+00 (00) 000 0000',
      ],
    );
```

Please see #41

**Comment by caseyryan** [comment]
Fixed in version 2.1.0
